src/models/Language.js

```diff
@@ -23,19 +47,36 @@ class Language extends Model {
 
     super(data);
 
-    this.#name = new MultiLangString(data.name);
+    Object.defineProperties(this, {
 
-    Object.defineProperty(this, `name`, {
-      configurable: true,
-      enumerable:   true,
-      get() {
-        return this.#name;
-      },
-      set(val) {
-        this.#name = new MultiLangString(val);
+      name: {
+        configurable: true,
+        enumerable:   true,
+        get() {
+          return this.#name;
+        },
+        set(val) {
+          this.#name = new MultiLangString(val);
+        },
       },
+
+      iso: {
+        configurable: true,
+        enumerable:   true,
+        get() {
+          return this.#iso;
+        },
+        set(val) {
+          this.#iso = new String(val);
+          // VALIDATE DATA
+          validateISOCode(this.#iso);
+        },
+      }
     });
 
+    this.name = data.name;
+    if ('iso' in data) this.iso = data.iso;
```

Comment:
```suggestion
    if (`iso` in data) this.iso = data.iso;
```

See [Template Literals Are Strictly Better Strings](https://ponyfoo.com/articles/template-literals-strictly-better-strings) for why we use template literals as the default whenever possible.